node/setup.go

```diff
@@ -560,6 +560,7 @@ var genesisDocHashKey = []byte("genesisDocHash")
 func LoadStateFromDBOrGenesisDocProvider(
 	stateDB dbm.DB,
 	genesisDocProvider GenesisDocProvider,
+	storeConfig *cfg.StorageConfig,
```

Comment:
Instead of passing the whole `storageConfig` just pass the `[]byte` for the hash since that's all you use ? Also in line 595 you check against the bytes anyways

Reply:
resolved